contracts/UsingWitnet.sol

```diff
@@ -6,84 +6,68 @@ pragma experimental ABIEncoderV2;
     witnet = WitnetRequestBoardInterface(_wrb);
   }
 
-  // Provides a convenient way for client contracts extending this to block the execution of the main logic of the
-  // contract until a particular request has been successfully resolved by Witnet
+  /// Provides a convenient way for client contracts extending this to block the execution of the main logic of the
+  /// contract until a particular request has been successfully resolved by Witnet
   modifier witnetRequestResolved(uint256 _id) {
     require(witnetCheckRequestResolved(_id), "Witnet request is not yet resolved by the Witnet network");
     _;
   }
 
-  /**
-   * @notice Check if a request has been resolved by Witnet.
-   * @dev Contracts depending on Witnet should not start their main business logic (e.g. receiving value from third.
-   * parties) before this method returns `true`.
-   * @param _id The unique identifier of a request that has been previously sent to the WitnetRequestBoard.
-   * @return A boolean telling if the request has been already resolved or not. Returns `false` if called after destroying the result (i.e. `destroyResult(uint256 _id).
-  **/
+  /// @notice Check if a request has been resolved by Witnet.
+  /// @dev Contracts depending on Witnet should not start their main business logic (e.g. receiving value from third.
+  /// parties) before this method returns `true`.
+  /// @param _id The unique identifier of a request that has been previously sent to the WitnetRequestBoard.
+  /// @return A boolean telling if the request has been already resolved or not. Returns `false` if called after destroying the result (i.e. `destroyResult(uint256 _id).
   function witnetCheckRequestResolved(uint256 _id) internal view returns (bool) {
     // If the result of the data request in Witnet is not the default, then it means that it has been reported as resolved.
     return witnet.readDrTxHash(_id) != 0;
   }
 
-  /**
-   * @notice Retrieves result of previously posted DR, and removes it from storage.
-   * @param _id The unique identifier of a previously posted data request.
-   * @return The result of the DR
-  **/
+  /// @notice Retrieves result of previously posted DR, and removes it from storage.
+  /// @param _id The unique identifier of a previously posted data request.
+  /// @return The result of the DR
   function witnetDestroyResult(uint256 _id) internal returns (WitnetData.Result memory) {
     return Witnet.resultFromCborBytes(witnet.destroyResult(_id));
   }
 
-  /**
-   * @notice Estimate the reward amount.
-   * @param _gasPrice The gas price for which we want to retrieve the estimation.
-   * @return The reward to be included for the given gas price.
-  **/
+  /// @notice Estimate the reward amount.
+  /// @param _gasPrice The gas price for which we want to retrieve the estimation.
+  /// @return The reward to be included for the given gas price.
   function witnetEstimateGasCost(uint256 _gasPrice) internal view returns (uint256) {
```

Comment:
```suggestion
  function _witnetEstimateGasCost(uint256 _gasPrice) internal view returns (uint256) {
```

Reply:
This change is postponed to `0.4.x` as it would imply breaking changes on the price feed examples, for instance.